.github/workflows/tests.yaml

```diff
@@ -0,0 +1,29 @@
+name: tests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  translations-check:
+    name: Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [18, 17, 16, 15, 14, 12, 10, 10.13]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install node
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Install dependencies
+        run: npm install
```

Comment:
The proper way is to run `npm ci` but this repo is missing `package-lock.json`.

Reply:
packages shouldn't have lockfiles :-) `npm ci` is only proper when there's a lockfile.